assets/js/googlesitekit/modules/create-info-store.js

```diff
@@ -36,18 +41,20 @@ const { createRegistrySelector } = Data;
  * @since 1.10.0
  * @private
  *
- * @param {string}  slug                  Slug of the module that the store is for.
- * @param {Object}  options               Options to consider for the store.
- * @param {number}  options.storeName     Store name to use.
- * @param {string}  options.adminPage     Store admin page. Default is 'googlesitekit-dashboard'.
- * @param {boolean} options.requiresSetup Store flag, for requires setup. Default is 'true'.
+ * @param {string}  slug                 Slug of the module that the store is for.
+ * @param {Object}  args                 Arguments to configure the store.
+ * @param {number}  args.storeName       Store name to use.
+ * @param {string}  [args.adminPage]     Optional. Store admin page. Default is 'googlesitekit-dashboard'.
+ * @param {boolean} [args.requiresSetup] Optional. Store flag, for requires setup. Default is 'true'.
  * @return {Object} The info store object.
  */
 export const createInfoStore = ( slug, {
 	storeName = undefined,
 	adminPage = 'googlesitekit-dashboard',
 	requiresSetup = true,
 } = {} ) => {
+	invariant( storeName, 'storeName is required.' );
+
 	const STORE_NAME = storeName || `modules/${ slug }`;
```

Comment:
* Since `storeName` is required now, this fallback should be removed.
* Also, let's update references below to use `storeName` rather than `STORE_NAME` to show that this is from the argument rather than an imported constant.